src/nvim/normal.c

```diff
@@ -2086,15 +2089,13 @@ static void op_colon(oparg_T *oap)
    */
 }
 
-/*
- * Handle the "g@" operator: call 'operatorfunc'.
- */
-static void op_function(oparg_T *oap)
+// Implements functionality of 'operatorfunc' and 'encodefunc'.
+static void call_function(const char *option_name, char_u *func, oparg_T *oap)
 {
   int save_virtual_op = virtual_op;
 
-  if (*p_opfunc == NUL)
-    EMSG(_("E774: 'operatorfunc' is empty"));
+  if (*func == NUL)
```

Comment:
You need to run clint: every time you change some line style errors are no longer ignore in it and surrounding lines so missing figure braces here will cause CI failure.